tools/tpm2_ptool.py

```diff
@@ -1030,27 +1074,211 @@ def __call__(self, args):
         args['no_init'] = True
         super(self.__class__, self).__call__(args)
 
-@commandlet("addkey")
-class AddKeyCommand(Command):
+class NewKeyCommandBase(Command):
     '''
-    Adds a key to a token within a tpm2-pkcs11 store.
+    creates a key to a token within a tpm2-pkcs11 store.
+    '''
+    def generate_options(self, group_parser):
```

Comment:
Do you need this, this should be inherited from class `Command`

Reply:
Now that I'm moving few options into the super class this will be needed here. Agree?